gensim/__init__.py

```diff
@@ -2,12 +2,27 @@
 of documents.
 """
 
-from gensim import parsing, corpora, matutils, interfaces, models, similarities, summarization, utils  # noqa:F401
 import logging
 
 __version__ = '3.8.1'
 
-
 logger = logging.getLogger('gensim')
 if len(logger.handlers) == 0:  # To ensure reload() doesn't add another one
     logger.addHandler(logging.NullHandler())
+
+#
+# Older versions of smart_open lack a NullHandler, meaning they can
+# inadvertedly call logging.basicConfig upon import.  This has been fixed in
```

Comment:
I'm not sure this is the case – see my investigation at https://github.com/RaRe-Technologies/smart_open/pull/476#issuecomment-614493217